pykdl_ros/src/pykdl_ros/__init__.py

```diff
@@ -41,6 +41,9 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self.__eq__(other)
 
+    def __hash__(self):
+        return hash((self.frame, self.header.stamp, self.header.frame_id))
```

Comment:
Shouldn't the __hash__() function only include the elements compared in __eq__()?

Reply:
The python documentation says the following: `The only required property is that objects which compare equal have the same hash value`. So objects with a different timestamp will compare equal, but have a different hash. So you are right.

I don't think we want to include the timestamp in the compare function. So removing the stamp from the hash function is the best way to go IMO. Do you agree @P-ict0?

Reply:
I think its the best option, yes